api/src/main/javadoc/doc-files/speclicense.html

```diff
@@ -42,9 +42,10 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
 
 <p>The notice is:</p>
 
-<p>&quot;Copyright &copy; 2018 Eclipse Foundation. This software or
-    document includes material copied from or derived from [title and URI
-    of the Eclipse Foundation specification document].&quot;</p>
+<p>&quot;Copyright &copy; 2018, 2020 Eclipse Foundation. This software or
+    document includes material copied from or derived from Jakarta™ Servlet
+    <a href="https://jakarta.ee/specifications/servlet/5.0/">
+        https://jakarta.ee/specifications/servlet/5.0/</a>.&quot;</p>
 
```

Comment:
Jakarta is a registered trademark. Again, see EL, JSP and WebSocket for correct examples.

Reply:
done